app.js

```diff
@@ -2,15 +2,18 @@ const express = require('express');
 const {getTopics} = require("../be-nc-news/controllers/topic-controllers")
 const app = express();
 const {readApi} = require("../be-nc-news/controllers/api-controllers");
-const { getArticleByArticleId } = require("../be-nc-news/controllers/article-controllers")
+const { getArticleByArticleId, getCommentsByArticleId } = require("../be-nc-news/controllers/article-controllers")
 
+app.use(express.json())
```

Comment:
This isn't needed at this stage! But as it is needed for the next task (POST), I'll let you off! :)